### PR TITLE
Enable support for all `Element`s

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Following https://chromium-review.googlesource.com/c/chromium/src/+/3934359, this PR updates the documentation to reflect support for all `Element`s.

This PR will be merged when the Finch experiment is rolled out 100% on stable channel. (Current target: Chrome 106)

@eladalon1983 Please review

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
@rachelandrew Please review

File: site/en/docs/web-platform/region-capture/index.md [hunk 1]
```diff
@@ -96,6 +93,10 @@ const cropTarget = await CropTarget.fromElement(mainContentArea);
 
 Unlike an `Element`, a `CropTarget` object is [serializable]. It can be passed to another document using [`Window.postMessage()`], for instance.
 
+{% Aside %}
+Prior to Chrome&nbsp;106, `CropTarget.fromElement(mainContentArea)` would reject if the type for `mainContentArea` was not `<div>` or iframe.
```
Comment (eladalon1983) [inline site/en/docs/web-platform/region-capture/index.md:97]: Shall we match `<div>` with `<iframe>`?

File: site/en/docs/web-platform/region-capture/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ authors:
   - beaufortfrancois
   - eladalon
 date: 2022-06-21
-updated: 2022-08-04
+updated: 2022-10-04
```
Comment (eladalon1983) [inline site/en/docs/web-platform/region-capture/index.md:10]: Let's not merge the PR until the rollout reaches 100%.

Comment (beaufortfrancois) [reply site/en/docs/web-platform/region-capture/index.md:10]: Yes. See https://github.com/GoogleChrome/developer.chrome.com/pull/3898#issue-1396197102

File: site/en/docs/web-platform/region-capture/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply site/en/docs/web-platform/region-capture/index.md:97]: I've used "iframe" for consistency in the article.

File: site/en/docs/web-platform/region-capture/index.md (hunk 2 above)
Comment (eladalon1983) [reply site/en/docs/web-platform/region-capture/index.md:10]: I'd missed that. Thanks for pointing it out.

**Comment by beaufortfrancois** [comment]
@eladalon1983 Can we merge this PR now?

**Comment by eladalon1983** [comment]
> @eladalon1983 Can we merge this PR now?

I think it's better to wait for the experiment to reach 100%.

**Comment by beaufortfrancois** [comment]
> > @eladalon1983 Can we merge this PR now?
> 
> I think it's better to wait for the experiment to reach 100%.

@eladalon1983 I believe we're good now. Do you confirm?

**Comment by eladalon1983** [comment]
> > > @eladalon1983 Can we merge this PR now?
> > 
> > 
> > I think it's better to wait for the experiment to reach 100%.
> 
> @eladalon1983 I believe we're good now. Do you confirm?

Almost. The experiment is at 50% atm.

**Comment by beaufortfrancois** [comment]
@eladalon1983 I believe we're good now for realz. Do you confirm?

**Comment by eladalon1983** [comment]
> @eladalon1983 I believe we're good now for realz. Do you confirm?

image

Let it rip.

**Comment by beaufortfrancois** [comment]
@rachelandrew Can you review?

**Pull request merged**